stylesheets/style.css

```
@@ -0,0 +1,67 @@
+body {
+    font-family: Arial;
+    background: #04c1e1;
+    margin: 0;
+    padding: 0;
+}
+
+h1, h2, h3 {
+    color: #1a0715;
+
+}
+
+.current-page {
+    font-weight: bold;
+}
+
+header {
+    grid-column: span 2;
+    text-align: center;
+}
+
+footer {
+    background: #f6ac0d;
+    margin-top: 1em;
+    padding: 1em 2em;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    grid-column: span 2;
+}
```

You can make your footer 'stick' to the end of the page by adding:

```suggestion
footer {
    background: #f6ac0d;
    margin-top: 1em;
    padding: 1em 2em;
    display: flex;
    flex-direction: column;
    justify-content: space-between;
    grid-column: span 2;
    position: fixed;
    left: 0;
    bottom: 0;
    width: 100%;
}
```

---

Here are some other ways to make the footer stick to the bottom of the page:

https://css-tricks.com/couple-takes-sticky-footer/